services/movie_session.py

```diff
@@ -42,3 +44,10 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    return [
+        row_and_seat for row_and_seat in MovieSession.objects.get(
+            id=movie_session_id).tickets.values("row", "seat")
+    ]
```

Comment:
There is no need to perform list comprehension, use typecasting instead
```suggestion
    return  list(MovieSession.objects.get(id=movie_session_id).tickets.values("row", "seat"))
```